### DISCUSSION TITLE
v1.0.0-beta.1

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#29] [Announcements]
## What's Changed
* Import legacy code - this imports the code with a fresh start from latest release by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/7
* Remove log license addon by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/8
* Apply all the un-released changes from legacy codeset by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/10
* Remove OS files and add them to .gitignore by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/11
* Update installation requirements check and links by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/14
* Add docker support by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/17
* Update the admin panel wiki links to go to the new URL by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/19
* Fleshing out the docker instructions by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/20
* Add docker install link by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/21
* Fix the error on the cron admin page by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/23
* Getting ready to make a release package by @jonyo in https://github.com/geodesicsolutions-community/geocore-community/pull/25

## New Contributors
* @jonyo made their first contribution in https://github.com/geodesicsolutions-community/geocore-community/pull/7

**Full Changelog**: https://github.com/geodesicsolutions-community/geocore-community/compare/v0.0.0...v1.0.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/geodesicsolutions-community/geocore-community/releases/tag/v1.0.0-beta.1'>v1.0.0-beta.1</a>.</em>

**Comment by jonyo** [comment]
And... I'm glad I called it beta, I just realized I did not do anything in the software itself to update the version number!  So there would be no visual way to see you are using the CE 1.0 version.

If someone has time and the skill, it would be nice to re-skin the footer, and update the version to 1.0.0.

I want to go back to using semver for versioning, that is the "gold standard" and makes it crystal clear what a new version has in it.

And I kinda want to start back over at 1.0 like I said.

That will make the update complicated because what happens when we catch up to the first versions in the updater, like 7.* I think?

So to combat that... I'm thinking, we tell people, if you are not already on 18.02.0 - update to that first.  Then the updates going forward will all be new and start at 1.0.  I'm just talking out loud here off top of my head...

So we probably need to skin the footer, if anyone has the time and know-how.  If no one is able I'll try to do it as I have time.

**Comment by vicos59** [comment]
1) I would suggest just starting at v20.0 just for continuity for current users and to avoid any future unforeseen issues.

2) This what appears for admin login window. Copyright also needs to be changed.

![bug 02](https://user-images.githubusercontent.com/6634041/152667886-864044ce-ba3e-4881-9b26-2aee33120c85.JPG)

3) There is 1 small issue with fresh install. The user_images directory is not created automatically. I was able to proceed after manually creating the dir.

![install bug 01](https://user-images.githubusercontent.com/6634041/152667947-cebe25f4-e286-43cc-8368-ebc391c7d87e.JPG)


4) Can you post a copy of the 18.02.0 final release somewhere? The last version I had downloaded was v17.x. I'll need to upgrade my 7.5.1 using this.

**Reply by jonyo** [reply]
> I would suggest just starting at v20.0 just for continuity for current users and to avoid any future unforeseen issues.

Fair enough.  Also it would not require more work to "start over" on versions.  We can start using semver again starting with 20.0.0.

I'll keep it named 1.0 beta until it is ready for a release then I'll switch over to 20 probably.

For 2 and 3 I made issues to fix them for beta 2, thanks!

> Can you post a copy of the 18.02.0 final release somewhere? The last version I had downloaded was v17.x. I'll need to upgrade my 7.5.1 using this.

Hmm, I could maybe email you something but it would be messy if you need PHP 7 compatibility.  You'd need the main files, the PHP7 patch, and the license patch.  Also I think even then it will be buggy (maybe) - the cron page in the admin may not work because of possible issue in the license patch.

If you are testing the CE and it gets to a solid state, you might consider just going strait to it.  I think we are close now to releasing the first "stable" version with you and other's help testing...  (as long as you are not picky with it still showing references to licensing in the admin)
